runtimes/bundles/org.jboss.tools.rsp.server.minishift/src/main/java/org/jboss/tools/rsp/server/minishift/servertype/impl/MinishiftServerTypes.java

```diff
@@ -30,7 +30,7 @@
 	public static final String RUNTIME_MINISHIFT_17_ID = "org.jboss.tools.openshift.cdk.server.runtime.type.minishift.17";
 	public static final String RUNTIME_CDK_30_ID = "org.jboss.tools.openshift.cdk.server.runtime.type.cdk.30";
 	public static final String RUNTIME_CDK_32_ID = "org.jboss.tools.openshift.cdk.server.runtime.type.cdk.32";
-	public static final String RUNTIME_CRC_10_ID = "org.jboss.tools.openshift.cdk.server.runtime.type.crc.100";
+	public static final String RUNTIME_CRC_10_ID = "org.jboss.tools.openshift.crc.server.runtime.type.crc.100";
```

Comment:
This is the runtime ID chosen in JBTools and used in stacks.yaml.  So we have to use it here.